gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RestErrorMapper.java

```diff
@@ -39,6 +41,19 @@ public static <T> Optional<ResponseEntity<T>> getResponse(
         .map(p -> ResponseEntity.of(p).build());
   }
 
+  public static ResponseEntity<Object> getValidationResponse(
```

Comment:
🔧 Validation, just like rejection, is probably rather controller-specific. Let's not generalize this prematurely yet. Plus, the error message is also not really correct since there was no command rejection. We never contacted the broker. Let's keep this simple and in the controller for now. Let's generalize when we encounter it a second time in the same manner.